data/json/speech.json

```diff
@@ -2017,7 +2149,7 @@
   },
   {
     "type": "speech",
-    "speaker": "foodperson_mask",
+    "speaker": [ "mon_eyebot", "mon_mi_go", "mon_mi_go_slaver" ],
```

Comment:
Looks like a copy-paste error, no?
`foodperson_mask` was changed to `mon_eyebot`
More of these below with `talking_doll`.